osu.Game.Rulesets.Osu/Skinning/Default/SliderBall.cs

```diff
@@ -152,8 +158,8 @@ protected override void Update()
             {
                 var otherKey = headCircleHitAction == OsuAction.RightButton ? OsuAction.LeftButton : OsuAction.RightButton;
 
-                // we can return to accepting all keys if the initial head circle key is the *only* key pressed, or all keys have been released.
-                if (actions?.Contains(otherKey) != true)
+                // we can return to accepting all other keys if they were released in the previous frame.
```

Comment:
the word "they" here is really throwing me off. i'm not sure what it's referring to.

Reply:
Hmm... This comment is very confusing in general I agree, because the surrounding code is pretty bad - in this case "all other keys" really just means one other key. Probably a refactoring job for another day.

"they" refers to "all other keys". Does it sound better as "we can start accepting any key once all other keys have been released in the previous frame"?

Reply:
After reading the surrounding code I grew to understand that, but it's best to explicitly state. What you said above sounds better.